packages/clerk-js/src/ui/common/RemoveResourceForm.tsx

```diff
@@ -34,6 +34,7 @@ export const RemoveResourceForm = withCardStateProvider((props: RemoveFormProps)
         <Text localizationKey={messageLine1} />
         <Text localizationKey={messageLine2} />
         <FormButtons
+          hasArrow={false}
```

Comment:
Are we deleting arrows from all screens? If yes, let's kill the prop. Or, if `hasArrow` is less frequent than `!hasArrow`, let's revert the default logic.

Reply:
Having said that, it would be preferable to replace FormButtons `hasArrow` property with a more composable, radix like approach and contract the arrow buttons whenever we have to.

Reply:
No we don't remove that from everywhere, it's just It less frequent as you said, so I reverted the logic and pushed the fix.